roles/update-and-install/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
We split system update from install of the software and found this to be useful in a few scenarios to de-couple those two actions. Our update system role:

https://github.com/theforeman/forklift/tree/master/roles/update_os_packages/tasks